finance-tracker/app/views/stocks/_list.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="row">
+  <div class="col-9">
+    <table class="table table-bordered table-hover">
+      <thead>
+        <tr>
+          <th scope="col">Ticker</th>
+          <th scope="col">Name</th>
+          <th scope="col">Price</th>
+          <th scope="col">Action</th>
+        </tr>
+      </thead>
+      <tbody>
+        <% @tracked_stocks.each do |stock| %>
+          <tr>
+            <th scope="row"><%= stock.ticker %></th>
+            <td><%= stock.name %></td>
```

Comment:
Why use `th` in `tr`?

Reply:
I think the tutorial wants it to be in bold 😅 I'll use the bold tag then.